docs/nodes/nodes-library/core-nodes/SSH/README.md

```diff
@@ -0,0 +1,68 @@
+---
+permalink: /nodes/n8n-nodes-base.ssh
+description: Learn how to use the SSH node in n8n
+---
+
+# SSH
+
+The SSH node is useful for executing commands via the Secure Shell Protocol.
+
+::: tip 🔑 Credential
+Authentication for the SSH node requires that you have a username and password for connecting to the remote server, or an SSH key configured for the server or service you are connecting to. See [Connecting to GitHub with SSH] for an example](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
+:::
+
+## Basic Operations
+
+- Execute a command
+- Download a file
+- Upload a file
+
+**Note:** To attach a file for upload, you will need to use an additional node such as the [Read Binary File](../../core-nodes/ReadBinaryFile/README.md) node or the [HTTP Request](../../core-nodes/HTTPRequest/README.md) node to pass the file as a data property.
+
+## Node Reference
+
+- ***Authentication:*** A dropdown list to choose between Password or Private Key authentication.
+- ***Resource:*** A dropdown list used to specify if you are executing a command or processing a file.
+- ***Operation:*** A dropdown list to select the action to be performed. When selecting **Command** as the ***Resource***, only **Execute** is available. For a **File** ***Resource*** you can select either **Upload** or **Download**.
+- ***Command:*** Only visible for **Command** resources. The command to execute on the remote machine.
+- ***Working Directory:*** Only visible for **Command** resources. The directory where the command should be executed.
+- ***Path:*** Only visible for **Download** operation on file resources. The path where the desired file is found.
+- ***Binary Property:*** Only visible for file resources. The name of the binary property which contains the data for the file to be uploaded.
+- ***Target Directory:*** Only visible for **Upload** operations on file resources. The directory to upload the file to. The name of the file does not need to be specified, it's taken from the binary data file name. To override this behavior, set the parameter **File Name** under options.
+
+
+## Example Usage
+
+This workflow allows you to remotely connect to your computer via SSH. This example usage workflow would use the following nodes.
```

Comment:
Can we have a more useful example? The example in its current state isn't helpful 🙂

Reply:
Also, please share the workflow on the workflow page and share the link here. This helps people to try out the workflow if they want to create it from scratch

Reply:
If they want to create it from scratch then sharing the workflow for them to copy defeats that purpose. Also, unless it is a particularly complex and useful example, I do not think each and every workflow should be uploaded to the Workflows page - it creates a mess, making it more difficult to find useful things on that page, without adding much value.

Reply:
Agree that a more useful example is needed. I have asked if there are any resources available internally I could use.

Reply:
I've updated my previous comment. We share the workflows on the workflow page for people who want to try out the example without creating the whole workflow.